app/views/roles/_form.html.erb

```diff
@@ -12,7 +12,26 @@
     <div class="tab-pane active" id="primary">
       <%= text_f f, :name, :class => @role.builtin? ? "disabled" : ""  %>
       <%= hidden_field_tag :original_role_id, @original_role_id if @cloned_role %>
+
+      <% if show_location_tab? || show_organization_tab? %>
+        <%= alert(:close => false, :class => 'alert-info', :header => '', :text => _("Changes to following associations will propagate to all inheriting filters ") +
```

Comment:
This line should not have trailing whitespace - if it's needed for the layout then it should be added outside of the translation string.